src/rustfmt/main.rs

```diff
@@ -413,6 +416,22 @@ fn format_string(input: String, opt: Opt) -> Result<i32> {
         verbosity: Verbosity::Quiet,
     };
     let report = rustfmt_nightly::format(Input::Text(input), &config, setting)?;
+
+    if report.has_errors() {
```

Comment:
I think that we don't want to emit errors in stdin mode by default, as it clutters the output.

Reply:
Fair enough, though wouldn't that result in rustfmt silently exiting with an error code in such circumstances?

Reply:
My bad, what I meant was when running in force mode, we do not want to emit the error report because both error reports and the formatting result will be printed. 

To summarize the behavior of the stdin mode when executed in conjunction with multiple flags,

```sh
rustfmt # Print error message
rustfmt --quiet # No error message
rustfmt --verbose # Print error message
rustfmt --force # No error message
rustfmt --force --quiet # No error message
rustfmt --force --verbose # Print error message
```

Reply:
Ah okay, that makes sense!